operator/pkg/ciliumendpointbatch/utils.go

```diff
@@ -0,0 +1,73 @@
+// Copyright 2021 Authors of Cilium
```

Comment:
Complete nit, but any chance we can call this file something other than util?

In this particular case it looks like there is a solid theme and purpose to everything in here. Can we just call it CEPCEBMap.go or something like that?

"util" files and packages have this annoying property of being very opaque, and especially if they share package definitions with other files I can't live without golsp, and no code reviewer I've seen lets me use it for reviews :)

Reply:
okay updated file name.